arangod/Graph/Enumerators/OneSidedEnumeratorInterface.h

```diff
@@ -73,23 +70,19 @@ class TraversalEnumerator {
       traverser::TraverserOptions::Order order,
       traverser::TraverserOptions::UniquenessLevel uniqueVertices,
       traverser::TraverserOptions::UniquenessLevel uniqueEdges,
-      arangodb::aql::QueryContext& query,
+      aql::QueryContext& query,
       typename Provider::Options&& baseProviderOptions,
-      arangodb::graph::PathValidatorOptions&& pathValidatorOptions,
-      arangodb::graph::OneSidedEnumeratorOptions&& enumeratorOptions,
-      bool useTracing) -> std::unique_ptr<TraversalEnumerator>;
+      graph::PathValidatorOptions&& pathValidatorOptions,
+      graph::OneSidedEnumeratorOptions&& enumeratorOptions, bool useTracing)
+      -> std::unique_ptr<TraversalEnumerator>;
 
-  using VertexRef = arangodb::velocypack::HashedStringRef;
-  TraversalEnumerator(){};
+  using VertexRef = velocypack::HashedStringRef;
+  TraversalEnumerator() {}
   virtual ~TraversalEnumerator() {}
 
-  // NOTE: keepPathStore is only required for 3.8 compatibility and
-  // can be removed in the version after 3.9
   virtual void clear(bool keepPathStore) = 0;
   [[nodiscard]] virtual bool isDone() const = 0;
 
-  // NOTE: keepPathStore is only required for 3.8 compatibility and
```

Comment:
This code comment turned out not to be true: SmartGraphEngine.cpp can call the function with a `keepPathStore` value of `true`.